src/utils/compose_ts.ts

```diff
@@ -708,7 +709,7 @@ export async function getServiceDirsFromComposition(
 			// remove './' prefix (or '.\\' on Windows)
 			dir = dir.startsWith(relPrefix) ? dir.slice(2) : dir;
 
-			serviceDirs[serviceName] = dir || '.';
+			serviceDirs[serviceName] = dir ?? '.';
```

Comment:
The typing does not indicate that this can be a non-string, so I would assume `dir` can be `''` and the goal was to replace that with `'.'`